### DISCUSSION TITLE
Compatibility of fluent bit 3.0.3 with opensearch 2.13.0

### DISCUSSION THREAD

**Discussion by amey-greenbone** [fluent/fluent-bit#8820] [Q&A]
I am running opensearch 2.13.0 in k8s version 1.28.x
I also configured fluentd with fluent/fluent-bit:3.0.3-amd64 version
All pods are running but fluentbit is throwing lot of exceptions.

**Comment by patrick-stephens** [comment]
What exceptions?
Are they covered in the docs already?

**Comment by patrick-stephens** [comment]
Currently we test against 1.3.0 by the looks of it for the self-hosted version: https://github.com/fluent/fluent-bit-ci/blob/761361b55e9b003842c4a1f0129673ea0f0b37fd/run-tests.sh#L42

Feel free to update that with a PR to verify.
